PWGHF/TableProducer/candidateSelectorD0.cxx

```diff
@@ -346,6 +359,15 @@ struct HfCandidateSelectorD0 {
           statusD0 = 0;
           statusD0bar = 0;
         }
+        registry.fill(HIST("DebugBdt/hBdtScore1VsStatus"), outputMl[0], statusD0);
+        registry.fill(HIST("DebugBdt/hBdtScore1VsStatus"), outputMl[0], statusD0bar);
+        registry.fill(HIST("DebugBdt/hBdtScore2VsStatus"), outputMl[1], statusD0);
+        registry.fill(HIST("DebugBdt/hBdtScore2VsStatus"), outputMl[1], statusD0bar);
+        registry.fill(HIST("DebugBdt/hBdtScore3VsStatus"), outputMl[2], statusD0);
+        registry.fill(HIST("DebugBdt/hBdtScore3VsStatus"), outputMl[2], statusD0bar);
+      }
+      if (statusD0 != 0 || statusD0bar != 0) {
+        registry.fill(HIST("hMassDmesonSel"), invMassD0ToPiK(candidate));
       }
```

Comment:
@lucamicheletti93 This causes a crash if `!applyMl` because `hMassDmesonSel` does not exist. I guess it should be moved inside the block just above. Can you please fix it?

Reply:
Or define the histogram outside the `applyMl` condition if meant for general debugging.
Actually, the path is wrong so the histogram does not exist in any case.